src/main/java/com/api/util/ApiSecurity/ApiSigning.java

```diff
@@ -261,7 +261,7 @@ public static PrivateKey getPrivateKeyFromKeyStore(String keystoreFileName, Stri
         try {
 
             try {
-                ks = KeyStore.getInstance("JKS");
+                ks = KeyStore.getInstance("PKCS12");
```

Comment:
Please help to check if changing to PKCS12 is necessary, java user preference normally in JKS format.